pkg/frontend/server.go

```diff
@@ -29,7 +29,7 @@ func ServeListen(backEndConfig *configs.BackEndConfig,
 		webRTCConfig:  webRTCConfig,
 	}
 
-	fs := http.FileServer(http.Dir("./public"))
+	fs := http.FileServer(http.Dir("./ui/build"))
```

Comment:
Replace this will custom stat. If the requested file does not exist, redirect to `/` (essentially causing `index.html` to resolve). Keywords: SPA, golang, catch-all.